src/router/index.js

```diff
@@ -1,19 +1,62 @@
-import React from "react";
-import {Routes, Route, Navigate} from "react-router-dom";
+import React, {useState} from "react";
+import {Routes, Route} from "react-router-dom";
 
 import GettingStarted from "Components/GettingStarted";
 import Components from "Components/Components";
-import Fabric from "Components/Fabric";
+import Home from "Components/Home";
+import Navbar from "Components/Navbar";
+import { ElvWalletClient } from "@eluvio/elv-wallet-client"; 
+
+
```

Comment:
Remove extra lines. Check throughout your changes. There generally shouldn't be more than one consecutive empty line 
Will also look into adding a linting rule for this